discovery-provider/default_config.ini

```diff
@@ -5,6 +5,7 @@ loglevel_flask = DEBUG
 ; set log level via command line in docker yml files instead
 ; loglevel_celery = INFO
 block_processing_window = 20
+block_processing_interval = 5
```

Comment:
amazing - its always unclear through our codebase when a var is in millisec or sec, i like when the names are `_interval_ms` but if you don't want to change it can you just throw a comment here indicating the units